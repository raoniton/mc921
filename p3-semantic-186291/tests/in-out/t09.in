class Main {
    public static void main(String[] args) {
        int i, count;
        int[] number = new int[25];

        print("How many elements are u going to enter (max=25)?");
        value = 23;

        print("Enter", count, "elements: ");
        for(i=0;i<count;i=i+1)
            number[i] = i*2;
    }
}