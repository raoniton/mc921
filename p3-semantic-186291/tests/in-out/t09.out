SemanticError: value is not defined @ 7:9
